src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
```

Comment:
According to the checklist, you don't need to name functions if you return them right away. The function 'getSum' is immediately returned, so you can omit its name and just return the function itself.